README.md

```diff
@@ -1 +1,230 @@
-# ballerina-aks-deployment
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/ballerina-aks-deployment.svg?branch=master)](https://travis-ci.org/ballerina-guides/ballerina-aks-deployment)
+
+# Deployment Ballerina with Azure Kubernetes Service
+
+[AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/) provides an easy to use hosted Kubernetes cluster service, which drastically reduces your time to setup your own infrastructure.
+
+> In this guide you will learn about building a Ballerina service and deploying it on Azure Kubernetes Service (AKS).
+
+The following are the sections available in this guide.
+
+- [What you'll build](#what-youll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Deployment](#deployment)
+- [Testing](#testing)
```

Comment:
We need to have an Observability section as well.

Reply:
As discussed offline, Observability is out of scope in this scenario.